java/README.md

```diff
@@ -55,3 +55,23 @@
      human-readable.
 
 9. Update the code to insert a document with the regular client. What happens?
+
+## KMIP
+
+**KMIP support in this guide is to be considered "best effort"**.
+
+This guide uses the [`direnv`](https://direnv.net/) tool to manage environment variables. 
+
+Update the `TRUSTSTORE_PASSWORD` environment variable in your `.envrc` file to your global Java TrustStore password.
+
+Run `direnv allow` to add the `TRUSTSTORE_PASSWORD` and `KEYSTORE_PASSWORD` environment variables.
+
+Run `source ./configure_certs.sh` to clone a tooling repository, install certificates, and start your KMIP
+server.
```

Comment:
I suggest emphasizing that the KMIP server is unstable and for testing only. PyKMIP must not be used in production environments.

Reply:
I changed this to "start your unstable kmip test server" and added AFAIK Github's [closest approximation to a warning admonition](https://github.com/github/markup/issues/887#issuecomment-418619745) saying not to use the production server.